src/config.go

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package main
+
+import (
+	"os"
+
+	"github.com/newrelic/infra-integrations-sdk/log"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	// configPathEnv is the environment variable used by the infrastructure-agent to pass extra configuration
+	// to integrations.
+	configPathEnv = "CONFIG_PATH"
+)
+
+// Config is the extra configuration that can be received by nri-cassandra from infrastructure agent.
+type Config struct {
+	// Include will define which cassandra metric definitions will be included.
+	Include QueryFilterConfig `yaml:"include"`
+	// Exclude specifies which nri-cassandra metric definitions will be excluded from collection.
+	Exclude QueryFilterConfig `yaml:"exclude"`
```

Comment:
Not sure if this should be a struct per see:
```
type Config struct {
	AttributeFilter Filter `yaml:"filter"`
}

type Filter struct {

	// Include will define which cassandra metric definitions will be included.
	Include QueryFilterConfig `yaml:"include"`
	// Exclude specifies which nri-cassandra metric definitions will be excluded from collection.
	Exclude QueryFilterConfig `yaml:"exclude"`
}
```

So that in the function below you can pass an AttributeFilter and not a generic "config"

Reply:
do you have a strong opinion about this? I would prefer the way it is right now because something like:
filter:
  - include
 in customer config, might cause confusion.

Reply:
not strong opinion 😄